### ISSUE TITLE
Dropbox link not working

### ISSUE THREAD

**Issue by songzeji** [qiime2/cancer-microbiome-intervention-tutorial#54]
Hi, first of all, thank you for making the very helpful tutorial! 

However, this dropbox link:

[https://www.dropbox.com/s/r5ag9d0lwlcg91n/tblcounts_asv_wide.csv?dl=1 ](url)

 in 04 - appendix convert-and-import-artifacts.ipynb is not working.

Would really appreciate it if you can fix it soon!

**Comment by ebolyen** [comment]
Not working for me either, I'm getting a server error (500) from dropbox, so this might be out of our control. @gregcaporaso anything change on your end?

**Comment by gregcaporaso** [comment]
It seems to be working for me - @songzeji, @ebolyen, could you try again?

**Comment by ebolyen** [comment]
Yep seems fine now, I think it was temporary issue with Dropbox servers.
